third_party/rust/chromium_crates_io/gnrt_config.toml

```diff
@@ -105,9 +105,20 @@ license_files = ['LICENSE']
 
 [crate.aes]
 extra_kv = { raw_gn = '''
+if (target_os == "android" && target_cpu == "arm64") {
+      deps -= [ "//brave/third_party/rust/cpufeatures/v0_2:lib" ]
+}
+''' }
+
+# gnrt conditional for this dep is merely if (target_cpu == "arm64")
+[crate.keccak]
```

Comment:
Would be nice to keep the crates configs in alphabetical order

Reply:
Fixed in fb45c252f3429fe09d4ba7ed3a41867b3c585fde